src/components/IconWithNetworkBadge.tsx

```diff
@@ -5,15 +5,45 @@ import { useSelector } from 'src/redux/hooks'
 import { networksIconSelector } from 'src/tokens/selectors'
 import { NetworkId } from 'src/transactions/types'
 
+export enum NetworkBadgeSize {
+  Large = 'Large',
+  Medium = 'Medium',
+}
+
+const NetworkBadgeSizeToStyle = {
```

Comment:
maybe leave a note to eventually support all these sizes? https://github.com/valora-inc/wallet/blob/5b92a5476b83455965731e3d98f5610ca10dc6dc/src/components/TokenIcon.tsx#L15